README.md

```diff
@@ -73,7 +73,10 @@ The project in this package uses the [Gradle](https://docs.gradle.org/current/us
 5. `./gradlew integTest -PnumNodes=3` launches a multi-node cluster with the asynchronous search plugin installed and runs all integ tests.
 6. `./gradlew integTest -Dtests.class=*AsynchronousSearchRestIT` runs a single integ class
 7.  `./gradlew integTest -Dtests.class=*AsynchronousSearchRestIT -Dtests.method="testSubmitWithRetainedResponse"` runs a single integ test method (remember to quote the test method name if it contains spaces)
-
+8. `./gradlew asynSearchCluster#mixedClusterTask -Dtests.security.manager=false` launches a cluster of three nodes of bwc version of OpenSearch with async search plugin and tests backwards compatibility by performing rolling upgrade of each node with the current version of OpenSearch with async search plugin.
```

Comment:
`8. mixedClusterTask` and `9. rollingUpgradeClusterTask` have the same description?